pybamm/expression_tree/array.py

```diff
@@ -128,7 +128,7 @@ def _jac(self, variable) -> pybamm.Matrix:
         jac = csr_matrix((self.size, variable.evaluation_array.count(True)))
         return pybamm.Matrix(jac)
 
-    def create_copy(self):
+    def create_copy(self, new_children=None):
         """See :meth:`pybamm.Symbol.new_copy()`."""
```

Comment:
do we need `new_copy` anymore. Presumably if `new_children=None` this function should be the same as `new_copy`?